file_prim/__init__.py

```
@@ -1060,6 +1060,77 @@ def draw(self, context):
+    )
+
+    use_aloc: BoolProperty(
+        name="Use Collision", description="Use a ALOC file on the chosen prim file"
+    )
+
+    aloc_filepath: StringProperty(
+        name="Collision Path",
+        description="Path to the Collision (ALOC) file",
+    )
+
+    def draw(self, context):
```

Ruff (0.14.8)

Unused method argument: `context`

(ARG002)